requirements.txt

```diff
@@ -1,7 +1,7 @@
-asyncio==3.4.3
-near-lake-framework==0.0.7
-requests==2.31.0
-toml==0.10.2
+asyncio>=3.4.3
+near-lake-framework>=0.0.8
+requests>=2.32.0
```

Comment:
This change also addresses dependabot alert: https://github.com/near/gas-station-event-indexer/security/dependabot/1